src/HaskellCI/GitHub.hs

```diff
@@ -394,23 +394,31 @@ makeGitHub _argv config@Config {..} prj jobs@JobVersions {..} = do
         { ghOn = GitHubOn
             { ghBranches = cfgOnlyBranches
             }
-        , ghJobs = Map.singleton "linux" GitHubJob
-            { ghjName      = "Haskell-CI Linux - GHC ${{ matrix.ghc }}"
-            , ghjRunsOn    = "ubuntu-18.04" -- TODO: use cfgUbuntu
-            , ghjSteps     = steps
-            , ghjContainer = Just "buildpack-deps:bionic" -- use cfgUbuntu?
-            , ghjMatrix    =
-                [ GitHubMatrixEntry
-                    { ghmeGhcVersion = v
-                    , ghmeAllowFailure =
-                           previewGHC cfgHeadHackage compiler
-                        || maybeGHC False (`C.withinRange` cfgAllowFailures) compiler
-                    }
-                | compiler@(GHC v) <- reverse $ toList versions
-                ]
-            }
+        , ghJobs = Map.fromList $ buildList $ do
+            item (mainJobName, GitHubJob
+                { ghjName            = "Haskell-CI Linux - GHC ${{ matrix.ghc }}"
+                , ghjRunsOn          = "ubuntu-18.04" -- TODO: use cfgUbuntu
+                , ghjNeeds           = []
+                , ghjSteps           = steps
+                , ghjIf              = Nothing -- TODO: Use cfgIrcIfInOriginRepo
```

Comment:
Should we file an issue about this remaining task?

Reply:
Ah, I think this was done in #436.

Reply:
Yeah, writing a patch felt easier than opening an issue.